book/src/proposals/booting-from-snapshot.md

```diff
@@ -0,0 +1,262 @@
+# Booting From a Snapshot
+
+## Problem
+
+A validator booting from a snapshot should confirm that they are caught up to
+the correct cluster before voting. There are two major risks if they do not:
+
+### Wrong Cluster
```

Comment:
Looks like the risk here is "accidentally locked out of the correct cluster", not "wrong cluster".